CLAUDE.md

```
@@ -0,0 +1,256 @@
+make ui    # Frontend only
+make api   # Backend only
+
+# View logs
+make logs
+
+# Stop all services
+make stop
+```
+
+**Ports**:
+- UI: http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)